Libva-Utils fails to compile with GCC 7.1.1

```
22:29:22,331 INFO  - test_va_api_init_terminate.cpp: In member function 'void VAAPI::VAAPIInitTerminate::doInitTerminate()':
22:29:22,331 INFO  - test_va_api_init_terminate.cpp:41:8: error: suggest explicit braces to avoid ambiguous 'else' [-Werror=dangling-else]
22:29:22,331 INFO  - if (vaDisplay)
22:29:22,331 INFO  - ^
```

--- a/test/test_va_api_fixture.cpp
+++ b/test/test_va_api_fixture.cpp
@@ -89,8 +89,9 @@ VADisplay VAAPIFixture::doInitialize()
     vaDisplay = getDisplay();
     EXPECT_TRUE(vaDisplay);
 
-    if (vaDisplay)
+    if (vaDisplay) {
         EXPECT_STATUS(vaInitialize(vaDisplay, &majorVersion, &minorVersion));
+    }
 
     return vaDisplay;
 }

--- a/test/test_va_api_init_terminate.cpp
+++ b/test/test_va_api_init_terminate.cpp
@@ -38,8 +38,9 @@ void VAAPIInitTerminate::doInitTerminate()
     vaDisplay = getDisplay();
     EXPECT_TRUE(vaDisplay);
 
-    if (vaDisplay)
+    if (vaDisplay) {
 	ASSERT_STATUS(vaInitialize(vaDisplay, &majorVersion, &minorVersion));
+    }
 
     EXPECT_EQ(VA_MAJOR_VERSION, majorVersion)
 	<< "Check installed driver version";

